azkaban-solo-server/src/main/java/azkaban/soloserver/AzkabanSingleServer.java

```diff
@@ -55,13 +62,39 @@ public AzkabanSingleServer(final AzkabanWebServer webServer,
 
   public static void main(final String[] args) {
     try {
+      initializeQuartzDb();
       start(args);
     } catch (final Exception e) {
       log.error("Failed to start single server. Shutting down.", e);
       System.exit(1);
     }
   }
 
+  private static void initializeQuartzDb() {
+    try {
+      Properties connectionProps = new Properties();
+      connectionProps.put("user", "sa");
+      connectionProps.put("password", "");
+      DriverManager.registerDriver((Driver) Class.forName("org.h2.Driver").getDeclaredConstructor().newInstance());
+      Connection conn = DriverManager.getConnection("jdbc:h2:./local/h2quartz", connectionProps);
+      String rawSql = IOUtils.toString(ClassLoader.getSystemClassLoader().getResource("quartz_schema.sql"));
+      String noComments = Arrays.stream(rawSql.split("\n")).map(x -> x.trim())
+          .filter(x -> !x.startsWith("--"))
+          .filter(x -> !x.isEmpty())
+          .reduce((a, b) -> a + "\n" + b).get();
+      for (String sql : noComments.split(";")) {
```

Comment:
I'll look at the runscript tool, but since we know what exact script we are executing we can easily verify that a simple character-level match is fine here.

Reply:
The main problem here is that I'd have to modify the build script to add h2 to the compile classpath, whereas I'm trying to make this fix minimally invasive. (It is currently part of the runtime classpath only)

Reply:
Sorry, I wasn't perfectly clear. I meant the statement approach, http://h2database.com/html/commands.html?highlight=runscript&search=runscript#runscript -- not the org.h2.tools.RunScript tool. The statement approach should not require build changes, should it?

Reply:
I'll test this out

Reply:
Implemented and tested, thanks